src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/ClimbableSurface.java

```diff
@@ -0,0 +1,51 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.tools;
```

Comment:
I'm not sure about the location of this class. We should have a package just for stuff like these.(There is also GrapplingHook class in listener package)

Reply:
I relocated the Surface and talisman settings into a dedicated package. But I'll be leaving the grappling hook one for now until we found a suitable place for it.